libs/common/img-util/src/processing.ts

```
@@ -125,3 +125,286 @@ export function findHistogramRange(
+    let bki: number
+    let bm, bmi: Int32Array
+
+    for (let i = 1, radiusi = radius - 1; i < radius; i++) {
+      blurKernel[radius + i] = blurKernel[radiusi] = bki = radiusi * radiusi
+      bm = blurMult[radius + i]
+      bmi = blurMult[radiusi--]
+      for (let j = 0; j < 256; j++) {
+        bm[j] = bmi[j] = bki * j
+      }
+    }
+    const bk = (blurKernel[radius] = radius * radius)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)